_stbt/stbt.conf

```diff
@@ -1,6 +1,10 @@
 [global]
 source_pipeline=videotestsrc is-live=true
-sink_pipeline=xvimagesink sync=false
+sink_pipeline=
```

Comment:
Isn't this missing `fakesink sync=false`?

Reply:
Not since https://github.com/stb-tester/stb-tester/pull/420/commits/5765354927b052e01cfd8379140081924689caba

Reply:
...plus 91ccfa1.